packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DateRangeFilter.test.tsx

```diff
@@ -198,3 +200,75 @@ it('should set filter and keep time from existing dates', () => {
     new Date(2021, 4, 3, 20, 30, 40, 500),
   ]);
 });
+
+it('should render with localized DatePicker', async () => {
```

Comment:
@gretanausedaite I created this unit test for DateRangeFilter. Do you think it's necessary creating another for the DatePickerInput? I'm concerned about duplicate.